business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/persps/ProjectLibraryPerspective.java

```diff
@@ -50,14 +52,23 @@ public boolean isDisplayed() {
     }
 
     private ImportRepositoryModal importProject() {
-        click(PROJECT_ACTIONS_BUTTON);
-        click(IMPORT_PROJECT_BUTTON);
+        if (Waits.isElementPresent(IMPORT_PROJECT_BUTTON)) {
```

Comment:
Is this false when some projects already available, or is this new `if` check rather for random failure protection.

Reply:
Yes. According to the latest changes if there is no project both buttons are present in the view:
![image](https://user-images.githubusercontent.com/4498639/67087240-cbefd800-f1a2-11e9-8245-21deb1bf77a1.png)
if there is already some project, the buttons are under the actions dropdown
![image](https://user-images.githubusercontent.com/4498639/67087289-eaee6a00-f1a2-11e9-8308-6e4fa94dada8.png)